cmake/sysbuild/partition_manager.cmake

```diff
@@ -366,32 +366,19 @@ function(update_runner)
   sysbuild_cache(CREATE APPLICATION ${RUNNER_IMAGE})
 endfunction()
 
-
 # APP is a special domain which is handled differently.
 # Remove it from the list.
 get_property(PM_DOMAINS GLOBAL PROPERTY PM_DOMAINS)
 list(REMOVE_ITEM PM_DOMAINS APP)
 
-## Check if current image is the dynamic partition in its domain.
-## I.E. it is the only partition without a statically configured size in this
-## domain. This is equivalent to the 'app' partition in the root domain.
-##
-## The dynamic partition is specified by the parent domain (i.e. the domain
-## which creates the current domain through 'create_domain_image()'.
-#if(DEFINED ${DOMAIN}_PM_DOMAIN_DYNAMIC_PARTITION
-#   AND "${IMAGE_NAME}" STREQUAL "${${DOMAIN}_PM_DOMAIN_DYNAMIC_PARTITION}"
-#)
-#  set(is_dynamic_partition_in_domain TRUE)
-#endif()
-
 get_property(PM_IMAGES GLOBAL PROPERTY PM_IMAGES)
 get_property(PM_SUBSYS_PREPROCESSED GLOBAL PROPERTY PM_SUBSYS_PREPROCESSED)
 
 # This file is executed once per domain.
 #
 # It will be executed if one of the following criteria is true for the
 # current image:
-# - It's a child image, and is the dynamic partition in the domain
+# - It's a secondary image, and is the dynamic partition in the domain
```

Comment:
minor nit.
I don't think we have introduced any concepts of primary or secondary images, but atm I have no better proposal and as this is a code comment, then it can stay.

Just wanted to mention.

Reply:
sub-image mayby? sub-domain?